testing/trino-product-tests-groups/src/test/java/io/trino/tests/product/TestTestGroups.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+import java.util.List;
+
+import static java.util.Locale.ENGLISH;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestTestGroups
+{
+    @Test
+    public void testGroupValues()
+            throws Exception
+    {
+        for (Field field : groupFields()) {
+            String name = field.getName();
+            String value = (String) field.get(null);
+
+            assertThat(name).as("Name of %s", field)
+                    .isUpperCase()
+                    .matches("[A-Z][_A-Z0-9]+")
```

Comment:
Why don't you like single char test group names :P

Reply:
i didn't mean to prohibit them, sorry!
it was just the easiest way to have _some_ validation.